src/me/raynes/fs.clj

```diff
@@ -350,7 +350,7 @@
   (default 10)."
   ([prefix]              (ephemeral-file prefix "" 10))
   ([prefix suffix]       (ephemeral-file prefix suffix 10))
-  ([prefix suffix tries] (when-let [created (temp-create prefix suffix tries create)]
+  ([prefix suffix tries] (when-let [^File created (temp-create prefix suffix tries create)]
```

Comment:
@harold I think it might make sense to add the type hint on the `temp-create` function instead.
So:
```
(defn- temp-create ^File [..])
```

Reply:
You got it, @borkdude!